src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write code here
+function salaryToNumber(salary) {
+  return +salary
+    .split('')
+    .filter((char) => '1234567890'.includes(char))
+    .join('');
```

Comment:
The `salaryToNumber` function currently converts the salary string to a number by filtering out non-numeric characters. This approach works but could be simplified using a regular expression to directly extract digits. Consider using `salary.replace(/\D/g, '')` to achieve the same result more efficiently.